### ISSUE TITLE
Summary2

### ISSUE THREAD

**Issue by Peiffap** [Gp2mv3/Syntheses#535]
Quelle serait la commande add.sh à faire pour ajouter un summary2? Je dois le faire manuellement ou il y a moyen d'automatiser?

**Comment by MartinBraquet** [comment]
Oui, c'est manuel, impossible avec add.sh car il match uniquement le mot "summary" :) .
Donc pour le moment, il faut copy-paste sans oublier de changer la variable TYPE dans le makefile en `summary2`.

**Comment by Peiffap** [comment]
Je dois faire ceci non?

`TYPE=summary
NUM=2
include ../eco-FSAB1803.mk`

**Comment by MartinBraquet** [comment]
Oh oui, juste ! Donc il faut juste ajouter NUM=2, et laisser le .tex en `eco-FSAB1803-summary.tex`.
